.travis.yml

```diff
@@ -9,12 +9,13 @@ before-install:
   
 script:
   - docker build --target test -t test .
-  - docker run -e TRELLO_KEY  -e TRELLO_TOKEN -e TRELLO_BOARD -e SECRET_KEY -e FLASK_APP='todo_app/app' -e FLASK_ENV='development' -e FLASK_SKIP_DOTENV='True' -e MONGO_USERNAME -e MONGO_PASS  -e MONGO_HOST -e DEFAULT_DATABASE -e LOGIN_DISABLED="True" test
+  - docker run -e TRELLO_KEY  -e TRELLO_TOKEN -e TRELLO_BOARD -e SECRET_KEY -e FLASK_APP='todo_app/app' -e FLASK_ENV='development' -e FLASK_SKIP_DOTENV='True' -e MONGO_CONNECTION_STRING -e LOGIN_DISABLED="True" test
```

Comment:
Given all the variables you cleaned out elsewhere - I suspect we can still remove some more from these test lines!